internals/schemas/translations.schema.json

```diff
@@ -0,0 +1,70 @@
+{
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "nl": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "translation": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "avgDisclaimerUrl": {
+              "type": "string",
+              "format": "uri"
+            },
+            "avgDisclaimer": {
+              "type": "string"
+            },
+            "consentToContactSharing": {
+              "type": "string"
+            },
+            "footer1": {
+              "type": "string"
+            },
+            "footer2": {
+              "type": "string"
+            },
+            "headerTitle": {
+              "type": "string"
+            },
+            "phoneNumber": {
+              "type": "string"
+            },
+            "shortTitle": {
+              "type": "string"
+            },
+            "siteTitle": {
+              "type": "string"
+            },
+            "smallHeaderTitle": {
+              "type": "string"
+            },
+            "title": {
+              "type": "string"
+            },
+            "urgentContactInfo": {
+              "type": "string"
+            }
```

Comment:
Would love to see description fields, for example would like to know for what `footer1` is being used.

Reply:
I considered adding descriptions, but, as the list of translations grows, so will the amount of descriptions. And for most strings it won't be necessary, superfluous even, because the key will/should be descriptive enough.

Reply:
I hear you, maybe we could otherwise use more descriptive names? `footer2` does not tell me anything, and if we will use `translations` outside the form I think it will get confusing really fast.

Reply:
Agree with jpoppe, more descriptive names are needed in order to keep it readable

Reply:
Very true. I'll rename those.